### DISCUSSION TITLE
[BUG]: Ring Size Scale not appearing in the shop manager but API

### DISCUSSION THREAD

**Discussion by yovasx2** [etsy/open-api#746] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

After setting Ring Size Scale, the API returns it but the value is not visible in the shop manager

### Steps to reproduce

Define the property for a listing:

```
curl --location --request PUT 'https://openapi.etsy.com/v3/application/shops/18976342/listings/1297365420/properties/54142602013' \
--header 'Content-Type: application/json' \
--data-raw '{
    "property_id": 54142602013,
    "property_name": "Ring size",
    "scale_id": 22,
    "scale_name": "FR",
    "value_ids": [],
    "values": [
        "40"
    ]
}'
```

response:
```
{
    "property_id": 54142602013,
    "property_name": "Ring size",
    "scale_id": 22,
    "scale_name": "FR",
    "value_ids": [
        629266885431
    ],
    "values": [
        "40"
    ]
}
```

Ensure the value was set using get properties:
```
curl --location --request GET 'https://openapi.etsy.com/v3/application/shops/18976342/listings/1297365420/properties' \
```
response:
```
...
  {
            "property_id": 54142602013,
            "property_name": "Ring size",
            "scale_id": 22,
            "scale_name": "FR",
            "value_ids": [
                629266885431
            ],
            "values": [
                "40"
            ]
        }
....
```

But the shop manager is showing blank:

<img width="1321" alt="Screen Shot 2022-09-24 at 12 21 40" src="https://user-images.githubusercontent.com/3578401/192110916-4f477656-b296-4260-a6b3-44ee62d299ad.png">

### Expected behavior

To visualize the value in the shop manager once the API returns it

### Additional context

Probs related with https://github.com/etsy/open-api/issues/502

**Comment by etsyachristensen** [comment]
@yovasx2 Out of curiosity, is the value in your system for the ring size a free text entry? Or is it a dropdown of values? The reason I ask is that I see that the value "40" was submitted and not the value id. I'm curious if you were enforcing a selection set on this one.

**Comment by yovasx2** [comment]
good point I need to check it, it's dropdown prepopulated, what is the name of those vs the free input?

**Comment by etsyachristensen** [comment]
apologies for the delays @yovasx2. We've been busy with the holidays. As we are working to close down the issues board in favor of the discussions board, I am going back through the outstanding issues. Is this still showing up as an issue?

**Comment by m4ntr4r4m4** [comment]
is this issue fixed?  because I have the same when I try to add length and width. it just returns **null** for **scale_name**
"property_name":"Width","scale_id":7,"scale_name":null,"value_ids":[733221261],"values":["8"]}]}
